gwsumm/html/bootstrap.py

```diff
@@ -109,9 +110,16 @@ def calendar(date, tag='a', class_='navbar-brand dropdown-toggle',
     page = markup.page()
     page.a('&laquo;', class_='navbar-brand step-back', title='Step back',
            onclick='stepDate(-1)')
+    attributekwargs = {'data-date': data_date,
+                       'data-date-format': 'dd-mm-yyyy',
+                       'data-viewmode': '%ss' % mode.name}
+    if highlighteddates is not None:
+        attributekwargs['highlight-dates'] = highlighteddates.replace('-', '')
+    else:
```

Comment:
@mattpitkin, I think the logic in lines 118-120 can be implemented in one line, something like:

```python
    elif highlightavailable:
        attributekwargs['highlight-available-dates'] = 'true'
```